src/pkg/logger/logger.go

```diff
@@ -110,7 +107,7 @@ func addFlags(fs *pflag.FlagSet, defaultFile string) {
 		&SensitiveInfoFV,
 		SensitiveInfoFN,
 		PIIPlainText,
-		fmt.Sprintf("set the format for sensitive info in logs to %s|%s|%s", PIIHash, PIIMask, PIIPlainText))
+		fmt.Sprintf("set the format for sensitive info in logs to %s|%s", PIIHash, PIIPlainText))
```

Comment:
Why do we have an option for this flag? Wouldn't it be easier to just have a `mask-sensitive-info` flag?

Reply:
Two reasons: one, it technically supports a third option that is hidden (mask, which replaces sensitive data with a flat `***`).  Two, I've been leaning towards flags with args over boolean flags for ease of future exensibility.  If we decide to formally add a new value here (maybe like letting users choose their hash seed `hash,seed:my_seed`) then we can extend the arguments rather than introduce a new flag.  Loosely held opinions here; so if you feel strongly about a boolean option, we can swap it out for that.

Reply:
I believe we can have the best of both worlds here.

- Default (no flag): Plain text
- No option `--mask-sensitive-info` flag: Current hashing (becomes the default masking option)
- Flag w/ options `--mask-sensitive-info <hash/mask>`: Whatever optional behavior we want to support

This supports a good default CLI UX while allowing us to support options in the future.

Reply:
Unfortunately, our CLI library requires categorizing flags as expecting arguments or not.  Calling a flag that expects arguments without any will either 1/ treat the next flag as an argument, or 2/ error (if no other flags are described).

We can get very close to that in the future, if needed, by maintaining both a common boolean flag for quick usage and clean UX (`--mask-sensitive-data`, in this case), plus second flag with more granular configuration for power usage (ex: `--mask-config mask:hash,seed:my-seed`).

I've noodled on doing the same for logging.  Maintain both the smaller `--log-level foo` and `--log-file bar` flags, plus a fully extensible flag for more granular config `--logging level:foo,file:bar,format:cli,minimal`.